pages/en/lb3/Third-party-login-using-Passport.md

```diff
@@ -297,6 +297,39 @@ For more information, see [Configuring providers.json](Configuring-providers.js
 }
 ```
 
+### Configuration in model-config.json
+
+loopback-component-passport comes with ApplicationCredential, UserCredential and UserIdentity models. These need to be configured in `model-config.json`.
+
+Add "./node_modules/loopback-component-passport/lib/models" to the sources array.
+
+```javascript
+"sources": [
+      "loopback/common/models",
+      "loopback/server/models",
+      "../common/models",
+      "./models",
+      "./node_modules/loopback-component-passport/lib/models"
```

Comment:
Shouldn't this be `../node_modules` because `model-config.json` is in the `server` directory, and `node_modules` is under the project root...?

Reply:
yes of course it should be!